notebooks/en/annotate_text_data_transformers_via_active_learning.ipynb

```diff
@@ -0,0 +1,2194 @@
+{
```

Comment:
<p>Please remove this, as this button is automatically generated for the notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>